interfaces/page-visibility.idl

```diff
@@ -4,12 +4,11 @@
 // See: https://w3c.github.io/page-visibility/
 
 enum VisibilityState {
-    "hidden",
-    "visible",
-    "prerender"
+  "hidden", "visible", "prerender"
```

Comment:
Prerender is "at risk", but I guess that's orthogonal to the refactor here. We'll have to remove it once (and if) we officially drop it from the spec.